spec/models/location_spec.rb

```diff
@@ -169,6 +169,83 @@
     expect(location).to be_valid
   end
 
+  describe '.online_booking_twilio_number' do
```

Comment:
Yeah, there's a mixture of it already in the spec, so I copied the one above. 😄 

I'll slip a commit in before mine tidying up the rest of them and amend mine to use `#`.